wiki/People/Osu!_Alumni/en.md

```diff
@@ -1,3 +1,6 @@
+osu! Alumni
```

Comment:
change folder to `osu! Alumni` (with lowercase `o`)?
- idk but `osu!alumni` (without a space, but with lowercase `a`)?